lib/config/presets/util.spec.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
Is this needed? I don't get a warning without it.

Reply:
removed, don't remember why added.